docs/administration/configuration/configuring-access/index.md

```diff
@@ -24,8 +24,62 @@ Helm value accordingly.
 
 ## User registration via Single Sign-On
 
-Handling of users with Single Sign-On (SSO) is also possible. Currently,
-this is only available for CERN deployments via [`components.reana_ui.cern_sso`](https://github.com/reanahub/reana/tree/master/helm/reana)
+Handling of users with Single Sign-On (SSO) is also possible. Currently SSO is supported for CERN as well as any third-party SSO provider which supports [Keycloak](https://www.keycloak.org/).
+
+### Keycloak Single Sign-On Configuration
+
+First of all, to integrate REANA with your Keycloak instance, you need to create a new client from the Keycloak's admin dashboard.
+There are many configuration options, but these are the minimal changes you have to make:
+
+- _Valid Redirect URIs_ should be set to `https://my-reana-hostname/api/oauth/authorized/keycloak/` (see Keycloak's [Access Settings](https://www.keycloak.org/docs/latest/server_admin/#access-settings))
+- _Client authentication_ should be enabled in order to get the _Client ID_ and the _Client Secret_ (see Keycloak's [Confidential client credentials](https://www.keycloak.org/docs/latest/server_admin/#_client-credentials))
```

Comment:
```suggestion
- _Client authentication_ should be enabled in order to get the client ID and the client secret (see Keycloak's [Confidential client credentials](https://www.keycloak.org/docs/latest/server_admin/#_client-credentials))
```
Again, just to be more coherent with the style of the rest of the documentation. I would still keep the italic around "Valid redirect URIs" and "Client authentication" since they refer to the Keycloak-specific option (maybe we could also surround them with quotes). What do you think?